live/lekce3/10.py

```diff
@@ -0,0 +1,37 @@
+import random
+from pprint import pprint 
+soubor = open("karty.txt","r")
+deck = [card for card in soubor]
+soubor.close()
+
+
+# print("zaprasene " + str(deck))
+# print("zaprasene {}".format(deck))
+pprint(f"zaprasene: {deck}")
```

Comment:
Dobré, kdyby se někdo oháněl že prasit je neslušné, můžeš říkat že to je "zaprášené" :)